variables.tf

```diff
@@ -9,6 +9,7 @@ variable "primary_container_definition" {
     ports                 = list(number)
     environment_variables = map(string)
     secrets               = map(string)
+    efs_volume_mounts     = list(map(string))
```

Comment:
any reason you used `map(string)` instead of an `object`?

Reply:
Done